packages/connect/CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ Use the persistent link connect.trezor.io/9 to access the latest stable version
 
 # 9.2.5-beta.1
 
+Improved reliability of popup handshake mechanism.
+Improved compatibility with webextensions using TrezorConnect inside offscreen (eg. Metamask).
+Cardano: support Conway certificates, increase minimum supported FW to 2.6.0 to clean up legacy code.
+Minor fixes in Connect Explorer and documentation.
```

Comment:
If we add the human friendly version shouldn't we get rid of the commit history like below?